sqswatcher/plugins/slurm.py

```diff
@@ -158,9 +155,10 @@ def removeHost(hostname, cluster_user):
 
 
 def restartMasterNodeSlurm():
-    slurmRestart = Path('/etc/systemd/system/slurmctld.service')
-    if slurmRestart.is_file():
-        command = 'sudo systemctl restart slurmctld.service'
+    if os.path.isfile('/etc/systemd/system/slurmctld.service'):
+        command = ['sudo', 'systemctl', 'restart', 'slurmctld.service']
+    elif os.path.isfile('/etc/systemd/system/slurmd.service'):
```

Comment:
This isn't needed.  This function is restartMasterNodeSlurm; the slurmd process never runs on the master node (ie, if this file exists, other things are broken).

Reply:
Ya, correct now.